app/Repositories/PaymentRepository.php

```diff
@@ -21,6 +22,14 @@ public function getAll($filter)
         return $this->model::orderBy('created_at', 'desc')->filter($filter)->paginate();
     }
 
+    public function store(array $data)
+    {
+        // dd($data);
+        if ($data["payment_type_id"] == 2) {
```

Comment:
WHat if repayment type id changes in the db?

Reply:
I still don't know how to check if it is a repayment